LayoutTests/imported/w3c/web-platform-tests/WebCryptoAPI/sign_verify/eddsa.https.any.worker-expected.txt

```diff
@@ -0,0 +1,39 @@
+
+PASS setup
+FAIL importVectorKeys step: Ed25519 verification assert_unreached: importVectorKeys failed for//// Ed25519SHA-512Curve25519. Message: ''Data provided to an operation does not meet requirements'' Reached unreachable code
```

Comment:
Why are we failing this test?

Reply:
Currently we are working with the raw format only, the other ones are not expected to work. Nevertheless, Ed25519-raw  are all passing